lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -278,6 +278,16 @@ class UsdUndoAddNewPrimAndSelectCommand : public Ufe::CompositeUndoableCommand
                 Ufe::GlobalSelection::get(), newSelection));
         }
     }
+
+#ifdef UFE_V4_FEATURES_AVAILABLE
+#if (UFE_PREVIEW_VERSION_NUM >= 4032)
+    std::string commandString() const override
+    {
+        return cmdsList().front()->commandString();
```

Comment:
Is it possible for cmdsList() to be empty?

Reply:
In that specific case, it always contains two commands (the first is the action, and the second is the selection).